autopush/websocket.py

```diff
@@ -295,15 +295,14 @@ def uaid(self, value):
 
     def set_connection_type(self, conn_type):
         """Set the connection type for the client"""
-        self.use_webpush = conn_type == "webpush"
+        self.use_webpush = True
```

Comment:
one more: kill self.use_webpush entirely, it's no longer used outside of this method. might as well kill the conn_type argument to this method too, have it hardcode webpush

Reply:
👍